versions.props

```diff
@@ -1,18 +1,21 @@
+# dependency-upgrader:OFF
```

Comment:
So the one downside of this is that we don't find out about deprecations early...

Can we include a comment saying "please only bump these if you need new features, as this is a very low level library and it will transitively upgrade the entire world"